bgpd/bgp_errors.c

```diff
@@ -302,5 +470,7 @@ static struct log_ref ferr_bgp_err[] = {
 
 void bgp_error_init(void)
 {
+	log_ref_add(ferr_bgp_warn);
 	log_ref_add(ferr_bgp_err);
+	log_ref_add(ferr_bgp_warn);
```

Comment:
Duplicate line?

Reply:
Doesn't look like a duplicate line to me